PyRDF/backend/Backend.py

```diff
@@ -52,6 +52,27 @@ def __init__(self, config={}):
 
         """
         self.config = config
+        self.includes = []
+
+    @staticmethod
+    def declare_headers(includes=[]):
```

Comment:
Why the default to []?

Reply:
Will remove this 👍

Reply:
One more thing here @etejedor is, if `includes` is gonna be in module-level, then I don't have to pass in a parameter here and I can just import it here directly right ?

Reply:
Mmm well, if this is the only place in `Backend` where you use `includes`, perhaps passing it as a parameter of `declare_headers` is fine.

Reply:
@etejedor , I can as well do an import inside the function `declare_headers` right ? `from .. import includes` ? Either way I have to do this import right ?

Reply:
Or like I said, just pass it as parameter of `declare_headers` and you do not need the import.

Reply:
@etejedor , you'll need the import to get the value of `includes` right ? Inside one of the backends, before execution, in order to get the value of `includes`, you'll have to import it right ?

Reply:
I do not get it now, but make the changes you find aproppriate, and I will re-review

Reply:
Ok @etejedor . Will do 👍